packages/calendar/src/props.ts

```diff
@@ -52,4 +52,5 @@ export const timeRangeDefaultProps = {
     dayRadius: 0,
     square: true,
     weekdayLegendOffset: 75,
+    firstDayOfWeek: 0,
```

Comment:
It could be nice to add a story for this or, ideally, to add it to the interactive doc.

Reply:
Added to the interactive docs.